src/language/printer.js

```diff
@@ -283,3 +283,13 @@ function printBlockString(value, isDescription) {
     ? `"""\n${isDescription ? escaped : indent(escaped)}\n"""`
     : `"""${escaped.replace(/"$/, '"\n')}"""`;
 }
+
+/**
+ * Prints variables one per line.
+ */
+function printVariableDefinitions(args) {
+  if (!args || args.length === 0) {
+    return '';
+  }
+  return '(\n' + indent(join(args, '\n')) + '\n)';
```

Comment:
Happy to see this function factored out to avoid duplication, but I think the additional new lines aren't helping legibility